cowprotocol/cowswap
# fix(price-impact): improve price impact loading

# Summary

Improve price impact loading

## Notes
- So far only really checked SWAP
- LIMIT orders might still not work as expected - working on it (either this PR or a follow up)
- There's a lot of noise as the fiat hooks interface changed to propagate the loading indicator flag

# To Test

1. On mainnet, pick sell and buy tokens and fill in the sell input
* Price impact will be calculated
* There should be a loading indicator to the right of the price impact
2. Change the amounts
* Price impact should be updated accordingly
* Loading indicator should be seen
